src/core/components/parameters/parameters.jsx

```diff
@@ -161,7 +161,10 @@ export default class Parameters extends Component {
               enabled={tryItOutEnabled}
               onCancelClick={this.props.onCancelClick}
               onTryoutClick={onTryoutClick}
-              onResetClick={() => oas3Actions.setRequestBodyValue({ value: undefined, pathMethod })}/>
+              onResetClick={() => {
```

Comment:
These two "internal" functions should be extracted a separate "onResetClick" prop function.

Reply:
Refactored into a prop